### PR TITLE
Support swagger file generation from schema

### PR DESCRIPTION
**Pull request by nati**
Closes: issue #32

### CODE REVIEW

**Comment by jbking** [comment]
Does this support schema actions?

**Comment by nati** [comment]
Hi 
I've added action support

File: docs/source/schema.rst [hunk 1]
```diff
@@ -663,4 +663,31 @@ The default for unspecified action is repeatble read.
...
+You can generate OpenAPI / Swagger file from gohan schema, so that you can
+afford swagger utility tools.
+
+.. code-block:: shell
+
+    gohan template --config-file etc/gohan.yaml
+
+    # or you can customize template file using
+
+    gohan template --config-file etc/gohan.yaml --template etc/templates/swagger.tmpl
```
Comment (jbking) [inline docs/source/schema.rst]: Found a bug https://github.com/cloudwan/gohan/issues/97

File: etc/templates/swagger.tmpl [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+                        "description": "unexpected error",
+                        "schema": {
+                        "$ref": "#/definitions/errorModel"
+                        }
+                    }
+                }
+            }
+        }{% for action in schema.Actions %},
+        "{{ schema.GetPluralURL() }}{{ action.Path | swagger_path }}": {
+            "post" : {
```
Comment (jbking) [inline etc/templates/swagger.tmpl]: Do you think `"{{ action.Method|lower }}"`?

File: docs/source/schema.rst (hunk 1 above)
Comment (nati) [inline docs/source/schema.rst]: ok i'll fix this in different patch.

File: etc/templates/swagger.tmpl (hunk 2 above)
Comment (nati) [inline etc/templates/swagger.tmpl]: got it

**Comment by nati** [comment]
ok it looks like there is no more comment on this.
I'll merge this.

**Pull request merged**